plutus-core/test/Evaluation/Machines.hs

```diff
@@ -44,56 +30,5 @@ testMachine machine eval =
 test_machines :: TestTree
 test_machines = testGroup
     "machines"
-    [ testMachine "CK"  $ evaluateCk  defBuiltinsRuntime
-    , testMachine "CEK" $ evaluateCek defBuiltinsRuntime
+    [ testMachine "CK" $ evaluateCk defBuiltinsRuntime
     ]
-
-testMemory :: ExMemoryUsage a => TestName -> a -> TestNested
-testMemory name = nestedGoldenVsText name . T.pack . show . memoryUsage
-
-test_memory :: TestTree
-test_memory =
-    runTestNestedIn ["test", "Evaluation", "Machines"]
-        .  testNested "Memory"
-        .  foldPlcFolderContents testNested testMemory testMemory
-        $  stdLib
-        <> examples
-
-testBudget :: TestName -> Plain Term DefaultUni DefaultFun -> TestNested
-testBudget name term =
-                       nestedGoldenVsText
-    name
-    (renderStrict $ layoutPretty defaultLayoutOptions {layoutPageWidth = AvailablePerLine maxBound 1.0} $
-        prettyPlcReadableDef $ runCek defBuiltinsRuntime (Restricting (ExRestrictingBudget (ExBudget 1000 1000))) term)
```

Comment:
I removed this and other tests from this file, but we still test this stuff in the untyped world.

Reply:
Are there some test output files we need to remove too?

Reply:
There's no `Golden` folder under `plutus/plutus-core/test/Evaluation` now (I did forget to generate new ones).